<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingDialog</class>
 <widget class="QDialog" name="settingDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>444</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Setting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/bwsetting.ico</normaloff>images/bwsetting.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>247</y>
     <width>320</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="charSetsBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>241</width>
     <height>269</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string notr="true">Changing the character set</string>
   </property>
   <widget class="QRadioButton" name="latcharsButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>36</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Lat</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="numberButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>74</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Number</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="ruscharsButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>112</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Rus</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="callsignsButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>150</y>
      <width>95</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Callsigns</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="usercharsButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>188</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">User</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="userText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>182</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QRadioButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>235</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">File</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="Tag" stdset="0">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="minCharsBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>16</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxCharsBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>61</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="groupsBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>106</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="speedBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>151</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="toneBox">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>196</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>500</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>700</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>23</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Min chars</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>68</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Max chars</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>114</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Groups</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>158</y>
     <width>98</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Speed (wpm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Tone (Hz)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>my_signal()</signal>
 </slots>
</ui>
